src/ClientHandle.h

```diff
@@ -369,6 +369,7 @@ class cClientHandle  // tolua_export
 	);
 	void HandleUnmount          (void);
 	void HandleUseEntity        (UInt32 a_TargetEntityID, bool a_IsLeftClick);
+	void HandleUseEntityAt      (UInt32 a_TargetEntityID, Vector3f a_TargetPos, bool a_IsLeftClick);
```

Comment:
`a_IsLeftClick` is a really confusing name since you can right click with the item in you left hand. I think `eHand` would be much better than a `bool` here.